resources/components/App.vue

```diff
@@ -12,16 +12,20 @@
 
 		<!-- Tabs container -->
 		<template v-if="showTabs">
-			<h2
-				v-i18n-html:machinevision-machineaidedtagging-tabs-heading
-				class="wbmad-suggested-tags-page-tabs-heading"
-			/>
+			<h2 v-i18n-html:machinevision-machineaidedtagging-tabs-heading
+				class="wbmad-suggested-tags-page-tabs-heading" />
 
 			<tabs v-on:tab-change="onTabChange">
-				<tab v-for="( tab, index ) in tabs"
```

Comment:
Since we have hard-coded title messages for each tab (and there are only 2 of them), looping over the tabs here didn't seem useful any longer. I defined computed properties to access the appropriate messages for cleaner template and easier testing (they need to me mocked since title is a required property of Tab).